aliasing/api/combat.py

```diff
@@ -225,6 +225,19 @@ def set_name(self, name: str):
             raise ValueError("Combatants must have a name.")
         self._combatant.name = str(name)
 
+    def set_group(self, group):
+        """
+        Sets the combatant's group
+
+        :param str group: The name of the group. None to remove from group.
+        :return: The combatant's new group, or None if the combatant was removed from a group.
+        :rtype: :class:`~aliasing.api.combat.SimpleGroup` or None   
+        """
+        if not (isinstance(group, str) or group is None):
+            raise ValueError('Group name must be a string or None.')
+
+        return self._combatant.set_group(group_name=group)
```

Comment:
This will return a normal `Group` rather than `SimpleGroup` if the combatant is added to a group